### PR TITLE
Introduce enzyme

### PR DESCRIPTION
**Pull request by busypeoples**
Added a reference test. This might need more work. 
Initially works in playground as this is Webpack independent.
Will need to adjust the Webpack configuration when introducing enzyme in client. #25

### CODE REVIEW

File: testHelper.js [hunk 1]
```diff
@@ -2,7 +2,34 @@ import chai from 'chai';
...
+});
+
+global.navigator = {
+  userAgent: 'node.js',
+};
+
+// documentRef = document;
+
+// prevent mocha tests from breaking when trying to require a less or css file
+require.extensions['.less'] = noop;
```
Comment (nikgraf) [inline testHelper.js]: I believe we won't use `less`. Can you remove the line?

**Comment by nikgraf** [comment]
sweet, super helpful!

**Comment by nikgraf** [comment]
is it ready to merge or should I still wait?

**Comment by busypeoples** [comment]
Removed the less line

**Comment by busypeoples** [comment]
Sure. This is ready to be merged. 
Should we add enzyme into the client part?

**Comment by busypeoples** [comment]
In that case, we'll have to adapt the Webpack config file or write a specific one for the tests. 
Check https://github.com/airbnb/enzyme/blob/master/docs/guides/webpack.md for further clarification

**Comment by nikgraf** [comment]
@busypeoples sounds good to me @ enzyme for the client

**Comment by nikgraf** [comment]
@busypeoples I sent you an invite to the organisation. then you should have access to all repos

**Comment by mxstbr** [comment]
Awesomeee!

**Pull request merged**